setup.py

```diff
@@ -40,6 +60,9 @@
     extra_compile_args.append('-UNDEBUG')
     sys.argv.remove("--debug")
 
+# avoid errors if SCIP is build with make:
+extra_compile_args.append('-DNO_CONFIG_HEADER')
```

Comment:
If we include this for all builds, we would have wrong information for cmake builds.

Reply:
O.k. I can apply this only if we use the makefile version.

Reply:
This is no only applied if a makefile version has been detected.